src/sql/parts/modelComponents/card.component.ts

```diff
@@ -52,15 +60,45 @@ export default class CardComponent extends ComponentBase implements IComponent,
 	// CSS-bound properties
 
 	public get label(): string {
-		return this.getPropertyOrDefault<sqlops.CardProperties, string>((props) => props.label, '');
+		return this.getPropertyOrDefault<CardProperties, string>((props) => props.label, '');
 	}
 
 	public get value(): string {
-		return this.getPropertyOrDefault<sqlops.CardProperties, string>((props) => props.value, '');
+		return this.getPropertyOrDefault<CardProperties, string>((props) => props.value, '');
+	}
+
+	public get actions(): ActionDescriptor[] {
+		return this.getPropertyOrDefault<CardProperties, ActionDescriptor[]>((props) => props.actions, []);
+	}
+
+	public hasStatus(): boolean {
+		let status = this.getPropertyOrDefault<CardProperties, StatusIndicator>((props) => props.status, StatusIndicator.None);
+		return status !== StatusIndicator.None;
+	}
+
+	public get statusColor(): string {
+		let status = this.getPropertyOrDefault<CardProperties, StatusIndicator>((props) => props.status, StatusIndicator.None);
+		switch(status) {
+			case StatusIndicator.Ok:
+				return 'green';
+			case StatusIndicator.Warning:
+				return 'orange';
+			case StatusIndicator.Error:
+				return 'red';
```

Comment:
Do these colors work fine on light, dark, and high-contrast themes? Otherwise we'll want to use theming for them

Reply:
Yes (for a given degree of yes). It's the same colors used in the Insight Details dialog to show if a DB backup is OK. So we've cleared this already, but I would love if we could make this themeable / configurable per-theme.

Light
![image](https://user-images.githubusercontent.com/10819925/39939191-47041722-550a-11e8-8ed9-5a3da5bd389d.png)

Dark
![image](https://user-images.githubusercontent.com/10819925/39939210-543567ca-550a-11e8-8027-c257d646cdd2.png)